recon.json

```
@@ -0,0 +1,632 @@
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "allowPoolAdmin(address,bool)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "approveDeposits(bytes16,uint128,uint128,address)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "approveRedeems(bytes16,uint128,uint128)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "claimDeposit(uint64,bytes16,uint128,bytes32)",
+        "actor": "admin",
+        "mode": "normal"
+      },
+      {
+        "signature": "claimRedeem(uint64,bytes16,uint128,bytes32)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "createAccount(uint32,bool)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "createHolding(bytes16,uint128,address,uint24)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "createPool(uint128,address)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "decreaseHolding(bytes16,uint128,address,uint128)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "execute(uint64,bytes[])",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "increaseHolding(bytes16,uint128,address,uint128)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "issueShares(bytes16,uint128,uint128)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "multicall(bytes[])",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "notifyPool(uint32)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "notifyShareClass(uint32,bytes16,bytes32)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "revokeShares(bytes16,uint128,uint128,address)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "setAccountMetadata(uint32,bytes)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "setHoldingAccountId(bytes16,uint128,uint32)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "setPoolMetadata(bytes)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "updateHolding(bytes16,uint128)",
+        "actor": "actor",
+        "mode": "normal"
+      },
+      {
+        "signature": "updateHoldingValuation(bytes16,uint128,address)",
+        "actor": "actor",
+        "mode": "normal"
+      }
+    ],
+    "separated": true
+  },
+  "out/SingleShareClass.sol/SingleShareClass.json": {
```

I suppose this file is auto-generated?

```suggestion
  "out/MultiShareClass.sol/MultiShareClass.json": {
```